bokeh/io.py

```diff
@@ -424,6 +431,38 @@ def push(session_id=None, url=None, app_path=None, document=None, state=None, io
     _push_to_server(session_id=session_id, url=url, app_path=app_path,
                     document=document, io_loop=io_loop)
 
+def push_notebook(document=None, state=None):
+    ''' Update the notebook with the data for the current document.
```

Comment:
this won't work or even make sense if you're using notebook with output_server right ? it should probably throw an exception saying "use push() instead" or maybe just do the right thing ...